packages/ordered-list/src/OrderedList.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { type StoryMetaType } from '@lg-tools/storybook-utils';
+
+import { Link } from '@leafygreen-ui/typography';
+
+import { ListItem, OrderedList } from '.';
+
+export const Basic = (args: { darkMode?: boolean }) => {
+  return (
+    <OrderedList {...args}>
+      <ListItem
+        title="Title"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna."
+      />
+      <ListItem
+        title="Title"
+        description={
+          <>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna.{' '}
+            <Link>Learn more.</Link>
+          </>
+        }
+      />
+      <ListItem
+        title="Title"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna."
+      />
+    </OrderedList>
+  );
+};
+
+const meta: StoryMetaType<typeof OrderedList> = {
```

Comment:
Nit: the meta is usually the first thing declared

Reply:
Even when it references an example?

Reply:
What's it referencing? The default he just references the name of a story not the story object itself.

Reply:
Updated